regcore/tests/db_django_models_tests.py

```diff
@@ -75,33 +75,33 @@ def setUp(self):
         self.dml = DMLayers()
 
     def test_get_404(self):
-        self.assertIsNone(self.dml.get('namnam', 'verver:lablab'))
+        self.assertIsNone(self.dml.get('namnam', 'cfr', 'verver:lablab'))
```

Comment:
Is `'verver:lablab'` a `doc_id` that could occur for a CFR layer? I thought the `doc_id` format for CFR layers was becoming, roughly, `version/section-subsection-...`. Speaking of which, a docstring in `models.py` or elsewhere laying out the format of `doc_id` might be useful.